config/database.yml.example

```diff
@@ -2,7 +2,11 @@ default: &default
   adapter: mysql2
   encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+<<<<<<< HEAD
```

Comment:
what is it??

Reply:
its in fixing process)

Reply:
always check it locally via git diff before pushing it